migrations/versions/3031ff741717_.py

```diff
@@ -0,0 +1,92 @@
+"""empty message
+
+Revision ID: 3031ff741717
+Revises: 1f6733ee5aa3
+Create Date: 2017-06-16 23:31:27.801256
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '3031ff741717'
+down_revision = '1f6733ee5aa3'
+
+from alembic import op
+import sqlalchemy as sa
+from app.models.event import Event
+from sqlalchemy.orm import sessionmaker
+
+Session = sessionmaker()
+
+
+def upgrade():
+    op.execute('ALTER TABLE events RENAME COLUMN ticket_include TO is_ticketing_enabled')
```

Comment:
Why are you executing raw queries for this ? Use alembic's methods to do the column rename please.